guides/common/modules/proc_creating-an-operating-system-for-debian.adoc

```diff
@@ -31,6 +31,12 @@ endif::[]
 +
 For more information, see xref:creating-partition-tables_{context}[].
 . On the *Templates* tab, select the `Preseed Finish` template as *Finish template*, the `Preseed` template as *Provisioning Template*, and the `Preseed PXELinux` template as *PXELinux template*.
+ifdef::orcharhino[]
+. On the *Parameters* tab, add the `or_client_repo_url` parameter, select the *string* type, and enter the value `\http://_{foreman-example-com}_/pulp/repos/_My_Organization_/Library/custom/{client-os}_Client/{client-os}_Client_{client-os-major}/`.
++
+If you want to provision hosts through {SmartProxyServers}, you have to use the `or_client_repo_path` parameter containing the *Published At* path of your content on {ProjectServer}.
+{SmartProxyServer} will add the FQDN of your {SmartProxyServer} to generate the URL based on the content source setting of your host.
+endif::[]
```

Comment:
Optional: This can be a snippet, but it can also stay as it is. Your call. Also, the format of the URL differs in the two instances, is that intentional?

Reply:
Thanks, fixed by adding a snippet and attribute (which I can overwrite in downstream for each OS differently)